### ISSUE TITLE
bug of md5 in aarch64 big-endian env

### ISSUE THREAD

**Issue by Liu-ErMeng** [openssl/openssl#20828]
a md5 bug in aarch64 big-endian env. Related tests are:
```
make V=1 TESTS='test_hmac' test
make V=1 TESTS='test_pbe' test
```

**Comment by nhorman** [comment]
Associated Pr is merged
marking as inactive, will be closed at the completion of 3.4 dev without further input

**Comment by t8m** [comment]
IMO we can close this as this is fixed.
